app/controllers/users_controller.rb

```diff
@@ -17,6 +17,6 @@ def create
   private
 
   def user_params
-    params.require(:user).permit(:username, :email, :password, :password_confirmation)
+    params.require(:user).permit(:email, :password)
```

Comment:
Hi @Euraldius - based on this [thoughtbot blog post on skipping password confirmation](https://robots.thoughtbot.com/email-and-password-confirmation-bias#no-password-confirmation), as well as our product preference, I added back the username functionality but will skip password confirmation.